### ISSUE TITLE
Like Wired, ForeignPolicy has an unblocked annoying interstitial 

### ISSUE THREAD

**Issue by weems** [brave/browser-laptop#1278]
On https://foreignpolicy.com/2016/04/05/obamas-most-dangerous-drone-tactic-is-here-to-stay/ there is an annoying interstitial which appears after a bit and is not blocked in Brave. 
![ForeignPolicy](https://cloud.githubusercontent.com/assets/470705/14375243/0f351830-fd25-11e5-8d85-52132058e859.png)

**Comment by diracdeltas** [comment]
Wired's interstitial was only shown to users who have adblockers; foreignpoicy's page is shown regardless of whether you are using an adblocker. I'm marking this as wontfix because they have a right to ask users to sign up to see their content.
